### PR TITLE
[samples] (Cross-reference workbooks) Adding flow steps for cross-reference sample

### PR DESCRIPTION
**Pull request by AlexJerabek**
This sample shows how to cross-reference data in two workbooks using scripts and Power Automate.

### CODE REVIEW

File: docs/resources/samples/excel-cross-reference.md [hunk 1]
```diff
@@ -123,28 +110,58 @@ function main(workbook: ExcelScript.Workbook, keys: string): string {
...
+1. This sample uses Outlook as the email client. You could use any email connector Power Automate supports. Add a **New step** that uses the **Office 365 Outlook** connector and the **Send and email (V2)** action. Use the following values for the action:
+    * **To**: Your test email account (or personal email)
+    * **Subject**: Event validation results
+    * **Body**: result (_dynamic content from **Run script 2**_)
 
-[Watch Sudhi Ramamurthy walk through this sample on YouTube](https://youtu.be/dVwqBf483qo").
+    :::image type="content" source="../../images/cross-reference-flow-3.png" alt-text="The completed Office 365 Outlook connector in Power Automate.":::
+1. Save the flow and try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations** and rows in **event-data**. Open **speaker-registrations** to see several highlighted cells where there are potential problems with the speaker registration listings.
```
Comment (alison-mk) [inline docs/resources/samples/excel-cross-reference.md:167]: Add file extensions? 
```suggestion
1. Save the flow and try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations.xlsx** and rows in **event-data.xlsx**. Open **speaker-registrations.xlsx** to see several highlighted cells where there are potential problems with the speaker registration listings.
```

Comment (AlexJerabek) [reply docs/resources/samples/excel-cross-reference.md:167]: Agreed, though I'm rephrase the try it out bit.

File: docs/resources/samples/excel-cross-reference.md [hunk 2]
```diff
@@ -123,28 +110,58 @@ function main(workbook: ExcelScript.Workbook, keys: string): string {
...
+1. This sample uses Outlook as the email client. You could use any email connector Power Automate supports. Add a **New step** that uses the **Office 365 Outlook** connector and the **Send and email (V2)** action. Use the following values for the action:
+    * **To**: Your test email account (or personal email)
+    * **Subject**: Event validation results
+    * **Body**: result (_dynamic content from **Run script 2**_)
 
-[Watch Sudhi Ramamurthy walk through this sample on YouTube](https://youtu.be/dVwqBf483qo").
+    :::image type="content" source="../../images/cross-reference-flow-3.png" alt-text="The completed Office 365 Outlook connector in Power Automate.":::
+1. Save the flow, then press **Test** to try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations.xlsx** and rows in **event-data.xlsx**. Open **speaker-registrations.xlsx** to see several highlighted cells where there are potential problems with the speaker registration listings.
```
Comment (alison-mk) [inline docs/resources/samples/excel-cross-reference.md:167]: 1. Favor "[select](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26472)" over "[press](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29040)" for inclusivity. 
2. When I tested this example, I had to click the "Save" button after entering the flow steps, and then go to "My flows" in the left hand pane, and then click the "Run" button next to this flow. Are you sure there's a Test button? I can't find it. 
```suggestion
1. Save the flow, then go to **My flows** and select **Run** next to this flow to try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations.xlsx** and rows in **event-data.xlsx**. Open **speaker-registrations.xlsx** to see several highlighted cells where there are potential problems with the speaker registration listings.
```

**Review by alison-mk — APPROVED** [review]
Approving with two suggestions (as one comment).

File: docs/resources/samples/excel-cross-reference.md (hunk 2 above)
Comment (AlexJerabek) [reply docs/resources/samples/excel-cross-reference.md]: Changed "press" and addressing the other comment as part of a subsequent PR.

**Pull request merged**